src/types/colorTable.cpp

```diff
@@ -0,0 +1,1045 @@
+    til::color{ 0xD7, 0x87, 0xFF },
+    til::color{ 0xD7, 0xAF, 0x00 },
+    til::color{ 0xD7, 0xAF, 0x5F },
+    til::color{ 0xD7, 0xAF, 0x87 },
+    til::color{ 0xD7, 0xAF, 0xAF },
+    til::color{ 0xD7, 0xAF, 0xD7 },
+    til::color{ 0xD7, 0xAF, 0xFF },
+    til::color{ 0xD7, 0xD7, 0x00 },
+    til::color{ 0xD7, 0xD7, 0x5F },
+    til::color{ 0xD7, 0xD7, 0x87 },
+    til::color{ 0xD7, 0xD7, 0xAF },
+    til::color{ 0xD7, 0xD7, 0xD7 },
+    til::color{ 0xD7, 0xD7, 0xFF },
+    til::color{ 0xD7, 0xFF, 0x00 },
+    til::color{ 0xD7, 0xFF, 0x5F },
+    til::color{ 0xD7, 0xFF, 0x87 },
+    til::color{ 0xD7, 0xFF, 0xAF },
+    til::color{ 0xD7, 0xFF, 0xD7 },
+    til::color{ 0xD7, 0xFF, 0xFF },
+    til::color{ 0xFF, 0x00, 0x00 },
+    til::color{ 0xFF, 0x00, 0x5F },
+    til::color{ 0xFF, 0x00, 0x87 },
+    til::color{ 0xFF, 0x00, 0xAF },
+    til::color{ 0xFF, 0x00, 0xD7 },
+    til::color{ 0xFF, 0x00, 0xFF },
+    til::color{ 0xFF, 0x5F, 0x00 },
+    til::color{ 0xFF, 0x5F, 0x5F },
+    til::color{ 0xFF, 0x5F, 0x87 },
+    til::color{ 0xFF, 0x5F, 0xAF },
+    til::color{ 0xFF, 0x5F, 0xD7 },
+    til::color{ 0xFF, 0x5F, 0xFF },
+    til::color{ 0xFF, 0x87, 0x00 },
+    til::color{ 0xFF, 0x87, 0x5F },
+    til::color{ 0xFF, 0x87, 0x87 },
+    til::color{ 0xFF, 0x87, 0xAF },
+    til::color{ 0xFF, 0x87, 0xD7 },
+    til::color{ 0xFF, 0x87, 0xFF },
+    til::color{ 0xFF, 0xAF, 0x00 },
+    til::color{ 0xFF, 0xAF, 0x5F },
+    til::color{ 0xFF, 0xAF, 0x87 },
+    til::color{ 0xFF, 0xAF, 0xAF },
+    til::color{ 0xFF, 0xAF, 0xD7 },
+    til::color{ 0xFF, 0xAF, 0xFF },
+    til::color{ 0xFF, 0xD7, 0x00 },
+    til::color{ 0xFF, 0xD7, 0x5F },
+    til::color{ 0xFF, 0xD7, 0x87 },
+    til::color{ 0xFF, 0xD7, 0xAF },
+    til::color{ 0xFF, 0xD7, 0xD7 },
+    til::color{ 0xFF, 0xD7, 0xFF },
+    til::color{ 0xFF, 0xFF, 0x00 },
+    til::color{ 0xFF, 0xFF, 0x5F },
+    til::color{ 0xFF, 0xFF, 0x87 },
+    til::color{ 0xFF, 0xFF, 0xAF },
+    til::color{ 0xFF, 0xFF, 0xD7 },
+    til::color{ 0xFF, 0xFF, 0xFF },
+    til::color{ 0x08, 0x08, 0x08 },
+    til::color{ 0x12, 0x12, 0x12 },
+    til::color{ 0x1C, 0x1C, 0x1C },
+    til::color{ 0x26, 0x26, 0x26 },
+    til::color{ 0x30, 0x30, 0x30 },
+    til::color{ 0x3A, 0x3A, 0x3A },
+    til::color{ 0x44, 0x44, 0x44 },
+    til::color{ 0x4E, 0x4E, 0x4E },
+    til::color{ 0x58, 0x58, 0x58 },
+    til::color{ 0x62, 0x62, 0x62 },
+    til::color{ 0x6C, 0x6C, 0x6C },
+    til::color{ 0x76, 0x76, 0x76 },
+    til::color{ 0x80, 0x80, 0x80 },
+    til::color{ 0x8A, 0x8A, 0x8A },
+    til::color{ 0x94, 0x94, 0x94 },
+    til::color{ 0x9E, 0x9E, 0x9E },
+    til::color{ 0xA8, 0xA8, 0xA8 },
+    til::color{ 0xB2, 0xB2, 0xB2 },
+    til::color{ 0xBC, 0xBC, 0xBC },
+    til::color{ 0xC6, 0xC6, 0xC6 },
+    til::color{ 0xD0, 0xD0, 0xD0 },
+    til::color{ 0xDA, 0xDA, 0xDA },
+    til::color{ 0xE4, 0xE4, 0xE4 },
+    til::color{ 0xEE, 0xEE, 0xEE },
+};
+
+static til::static_map xorgAppColorTable{

```

Comment:
I was wondering if Trie like https://github.com/smilingthax/cttrie could help reduce the binary & memory footprint of this